packages/puppeteer-core/src/common/ConnectOptions.ts

```diff
@@ -49,7 +49,9 @@ export interface BrowserConnectOptions {
   _isPageTarget?: IsPageTargetCallback;
 
   /**
-   * @defaultValue 'cdp'
+   * @defaultValue 'cdp' when launching Chrome and 'webDriverBiDi' when
+   * launching Firefox. When connecting to a running browser, defaults
+   * to 'cdp'.
```

Comment:
Let's simplify it something like:
```suggestion
   * @defaultValue Determined at run time:
   * Launching Chrome - 'cdp'.
   * Launching Firefox - 'webDriverBiDi'.
   * Connecting to browser - 'cdp'
```

Reply:
I do not think we would render the list correctly?

Reply:
```suggestion
   * @defaultValue Determined at run time:
   *
   * - Launching Chrome - 'cdp'.
   *
   * - Launching Firefox - 'webDriverBiDi'.
   *
   * - Connecting to a browser - 'cdp'.
   *
```

Reply:
![image](https://github.com/user-attachments/assets/c95f48e5-1f2b-459d-a3c9-4ec09b49830e)